Cargo.toml

```diff
@@ -60,6 +60,7 @@ indoc = "2.0.4"
 insta = { version = "1.38.0", features = ["filters"] }
 itertools = "0.12.1"
 libc = { version = "0.2.153" }
+libssh2-sys = { version = "0.3.0" }
```

Comment:
This dependency could be made Windows-only by moving it to a `[target.'cfg(windows)'.dependencies]` section.

Reply:
This isn't in the dependencies key; it's in the workspace.dependencies key. I tried target.'cfg(windows)'.workspace.dependencies and it said it was not allowed.